tests/plugins/emoji/basic.js

```diff
@@ -0,0 +1,71 @@
+/* bender-tags: emoji */
+/* bender-ckeditor-plugins: emoji,toolbar,stylescombo,format,clipboard */
+
+( function() {
+	'use strict';
+
+
+	bender.editors = {
+		inline: {
+			creator: 'inline',
+			name: 'inline',
+			startupData: '<p>foo:grinning_face:bar</p>'
+		},
+		classic: {
+			creator: 'replace',
+			name: 'classic',
+			startupData: '<p>foo:grinning_face:bar</p>'
+		}
+	};
+
+	var tests = {
+		setUp: function() {
+			if ( CKEDITOR.env.ie && CKEDITOR.env.version < 9 ) {
+				assert.ignore();
+			}
+		},
+		'test emoji is converted during editor creation': function( editor ) {
```

Comment:
Please, add a single line of spacing between `tests` object members. This will improve readability.